api/inc/page_allocator_exports.h

```diff
@@ -34,7 +44,7 @@
     struct { \
         size_t page_size; \
         size_t page_count; \
-        void* page_origins[count]; \
+        void *page_origins[count]; \
```

Comment:
Can this go directly [here](https://github.com/ARMmbed/uvisor/pull/223/commits/bf52c4f1d702b9cb3afdae542a5a35bce519f3a9#diff-c1b58e0b36c6333e689bf1dae14a5044R37)?